NetKVM/Common/ParaNdis_Common.cpp

```diff
@@ -1098,7 +1112,23 @@ static NDIS_STATUS ParaNdis_VirtIONetInit(PARANDIS_ADAPTER *pContext)
     DEBUG_ENTRY(0);
     UINT i;
     USHORT nVirtIOQueues = pContext->nHardwareQueues * 2 + 2;
-
+#if MAX_FRAGMENTS_IN_ONE_NB
```

Comment:
style: guessing we want the new line that used to be on 1101

Reply:
ok, I will add new line here